artifactory/commands/utils/questionnaire.go

```diff
@@ -276,3 +276,13 @@ func OptionalKeyCallback(iq *InteractiveQuestionnaire, key string) (value string
 	}
 	return value, err
 }
+
+func IsValideParams(key string, value interface{}, writersMap map[string]AnswerWriter) bool {
```

Comment:
I suggest rename this function to "validateMapEntry"
and let the return value be an error with the correct msg: "unknown-key" or "non-string-value"